dotnet-install.cmd

```diff
@@ -11,7 +11,7 @@ setlocal
     exit /b 1
   )
 
-  set DotNet_Path=%~dp0tools\dotnet\%DotNet_Version%
+  set DotNet_Path=%~dp0packages\dotnet\%DotNet_Version%
```

Comment:
>packages [](start = 23, length = 8)

Please keep the DotNet Cli under the tools. Packages folder should be used for restored packages.